tensorflow_text/python/ops/wordpiece_tokenizer_test.py

```diff
@@ -305,6 +305,32 @@ class WordpieceOpTest(ragged_test_util.RaggedTensorTestCase,
           vocab=_DEATH_VOCAB,
           max_bytes_per_word=40,
       ),
+      # Test not splitting out unknown characters.
+      # (p and ! are unknown)
+      dict(
+          tokens=[[b"nap", b"hello!me"]],
+          expected_subwords=[[[b"[UNK]"], [b"[UNK]"]]],
+          unknown_token="[UNK]",
+          vocab=_ENGLISH_VOCAB,
+      ),
+      # Test splitting out unknown characters.
+      dict(
+          tokens=[[b"nap", b"hello!me"]],
```

Comment:
I was thinking more about this test case and wondering if we should support multiple unknown characters instead of single unknown character. For example, for the test case:

["hello!!!!!me"] -> ["hello", "##[UNK]", "##[UNK]", "##[UNK]", "##[UNK]", "##[UNK]", me"]

should this instead collapse to  ["hello", "##[UNK]",  me"]? WDYT?

Reply:
I don't know, it might be useful, but I suspect the first output is better for sequence models- it keeps the information about the number of characters. And if you don't use the unk token, and use oov bucketing, the model has a chance of also learning a representation for the individual character.

For our internal use we need the current behaviour. It's also in line with what sentencepiece does so I suspect others would expect this behaviour too.

Could this behaviour be considered separately to this PR, and potentially be added as an option to the op?

Reply:
yes, let's consider this in a separate PR.